tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -2507,6 +2501,13 @@ public function testPendingToCompleteContribution() {
     );
     $this->setUpPendingContribution($this->_ids['price_field_value'][0]);
     $this->callAPISuccess('membership', 'getsingle', array('id' => $this->_ids['membership']));
+    // Case 1: Assert that Membership Signup Activity is created on Pending (Pay later) to Completed Contribution via backoffice
```

Comment:
Remove words pay later - the contribution created is (specifically) not a pay later one